hockey-score/src/app/shared/header/header.component.ts

```diff
@@ -7,18 +8,19 @@ import { Component, OnInit } from '@angular/core';
 })
 export class HeaderComponent implements OnInit {
 
-  counter: number = 0;
-  constructor() { }
+  isLogin: boolean;
+
+  constructor(private readonly authService: AuthService) {
+    this.isLogin = false;
```

Comment:
it looks like it is unnecessarily
you 've defined property isLogin: boolean; until value is assigned it is undefined.